### PR TITLE
added live file rotation detection

### PR DESCRIPTION
**Pull request by djay**
adds ability to keep reading from a file, without closing it, when the file is rotated.

### CODE REVIEW

File: pygtail/core.py [hunk 1]
```diff
@@ -59,6 +59,9 @@ def __init__(self, filename, offset_file=None, paranoid=False, copytruncate=Fals
                 # Look for the rotated file and process that if we find it.
                 self._rotated_logfile = self._determine_rotated_logfile()
 
+        # test that this works, allow exceptions to propergate at this point
```
Comment (Gopalvenu) [inline pygtail/core.py]: Hi,

pygtail module not handle the ams auto rotation log files. Kindly provide the fix for this.

Thanks,
Gopal.

**Comment by Gopalvenu** [comment]
Hi,

pygtail module not handle the ams auto rotation log files. Kindly provide the fix for this.

Thanks,
Gopal.

**Comment by lucasdiedrich** [comment]
Is there any change of this PR be committed any time soon?

Thanks.

**Comment by bgreenlee** [comment]
Sorry, I will look at this again when I'm back from sabbatical in early September

**Comment by djay** [comment]
Sorry it's been so long and I didn't write all that code that I can't really answer these review questions

**Comment by bgreenlee** [comment]
Fixed in #37

**Pull request closed without merging**